magento/magento2
# Increase minimum password length for admin user

### Description (*)
PCI 4.0 requires a minimal password length of 12 characters, so it has to be changed from 7 to 12.

### Related Pull Requests
https://github.com/magento-gl/magento2ee/pull/538
https://github.com/magento-gl/magento2b2b/pull/254


### Manual testing scenarios (*)

1. Change password of an existing admin account.
2. It should fail to change the password if less than 12 characters are used.

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#39327: Increase minimum password length for admin user
